xml/System.Net/SecurityProtocolType.xml

```diff
@@ -93,7 +84,7 @@
         <ReturnType>System.Net.SecurityProtocolType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Specifies the system default security protocol as defined by [Schannel](https://go.microsoft.com/fwlink/?linkid=833507).</summary>
+        <summary>Allows the operating system to choose the best protocol to use, and to block protocols that are not secure. Unless your app has a specific reason not to, you should use this field.</summary>
```

Comment:
"you should use this field".  It is not a "field".  This is an enumeration value from the `SecurityProtocolType` enumeration.

So, that phrase would be better as:

"you should select this enum value"

Reply:
docs.microsoft.com calls them "fields".

Reply:
That should be changed then.  "Fields" are within a class object.  They usually aren't something that is exposed publicly.

"Properties" on an object are something that can be "get" or "set".

But this documentation above is simply describing the different values from an enumeration type (SecurityProtocolType). In that case, "fields" does not seem right.

@terrajobst Can you comment on the proper .NET terminology for these things?

Reply:
Hey @mairaw. How are we referring to members of enums? Are we calling them fields?

@davidsh technically, enums are encoded in metadata very similar to classes/structs and their member are just public const fields. So either terminology would work for me.

Reply:
We do refer to them as fields but I think in this context I prefer value. 

However, this guidance is very light. Probably something more direct probably works better. 
"Unless your app has a specific reason not to, use this value."

Reply:
I'm happy to change the word to "value" if that's the consensus, but it makes no sense to use a different term from the title that's right above it (and is on every Enum topic). I was hoping that the decision would be to leave it consistent but spin off another thread (probably private) to discuss whether or not to comprehensively lose the internals-seeming term "field" and move (consistently) to more usual language. But as I say I'll change it here (to be inconsistent) for the sake of moving on.

Reply:
docs.microsoft.com lists enum members as fields, though MSDN simply described them as members; AFAIK, the change was simply due to an MSDN customization that wasn't ported to docs. They are fields, but we've generally not referred to them as fields, since that's an implementation detail, and most developers don't think of them as fields. So I'd say the guidance is to refer to a member as an enumeration member (which is generally what I do) or an enumeration value. I'd avoid field.
Here, since there's no question that the thing being documented is the enumeration member, I'd agree with @mairaw that the sentence best reads, "Unless your app has a specific reason not to, use this value."

Reply:
I would say that that heading needs to be changed, then. That heading essentially says "here are some fields", but writers are being asked to call them something else *right below that heading*.